app/default/account_routes.py

```diff
@@ -19,52 +22,96 @@
 account_bp = Blueprint("account_routes", __name__, template_folder="templates")
 
 
-def build_application_data_for_display(applications: list[ApplicationSummary]):
+def get_visible_funds(visible_fund_short_name):
+    """
+    Returns a list of funds matching the supplied short name
 
+    :param visible_fund_short_name: short name to look for
+    """
+    all_funds = get_all_funds()
+
+    if visible_fund_short_name:
+        funds_to_show = [
```

Comment:
one to check with @srh-sloan when she's back?

Reply:
Yeah I think you're right we would never match multiple funds here, I think I did it as the call to `get_all_funds()` is above it so we are filtering it. But looking at tit now this would seem like a better option:
`if visible_fund_short_name:
    funds_to_show = get_fund_by_short_name()
else:
    funds_to_show = get_all_funds()`

Then line 43 onwards. Benefit of a fresh pair of eyes :)